### DISCUSSION TITLE
Цветокоррекция в связке со сменой дня и ночи

### DISCUSSION THREAD

**Discussion by Xavely** [octothorp-team/public-dobrograd#537] [Предложения]
Приветствую!
Garry's Mod поддерживает цветокоррекцию игры в реальном времени, а точнее во вкладке "Пост-обработка > Шейдеры", и перейдя по этому пути мы получим доступ к одной из многочисленных опций, а именно как я ранее и обозначил - цветокоррекция.

Мое предложение заключается в том что бы с помощью этой самой цветокоррекции сделать ночи более тёмными и выкалывающими глаза при попытке приглядеться (Я имею ввиду не механику приглядеться а реальность).

Как это примерно будет работать на уровне машины:

```
Запрашиваю у atmos информацию о времени
Если информация выше этого значения: "Вставьте время" и не выше "Вставьте время" - правда - перейти к включению фильтра.
Если информация выше значения: "Вставьте время" - правда - перейти к выключению фильтра.
```

Я постарался объяснить как мог, Челог вроде бы немного уже поработал с фильтрами при настройке Киберграда (Он же как то сделал авто-наложение синеватого фильтра).

**Comment by Xavely** [comment]
@chelog

**Comment by LENFYS** [comment]
За
